src/arkode/arkode_mristep.c

```diff
@@ -2325,9 +3689,8 @@ int mriStep_ComputeInnerForcing(SUNDIALS_MAYBE_UNUSED ARKodeMem ark_mem,
 }
 
 /*---------------------------------------------------------------
-  Compute/return the 'effective' RK coefficients for a 'nofast'
-  stage.  It is assumed that the array 'A' has already been
-  allocated to have length MRIC->stages.
+  Compute/return the effective RK coefficients for a "nofast"
+  stage.  We may assume that "A" has already been allocated.
   ---------------------------------------------------------------*/
 
 int mriStep_RKCoeffs(MRIStepCoupling MRIC, int is, int* stage_map,
```

Comment:
I also prefer this here, since the purpose of the routine is purely to support how we have implemented  `mriStep_StageERKNoFast` and `mriStep_StageDIRKNoFast`.  If those were implemented differently, then the contents of `mriStep_RKCoeffs` could be changed accordingly, even though the coupling coefficient table (and its properties) would remain unchanged.